### PR TITLE
Added recommended training and reading resource

### PR DESCRIPTION
**Pull request by SteveLee1287**
Added Windows Powershell Cookbook

# PR Summary

## PR Checklist

- [ ] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [ ] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [ ] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [ ] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

File: docs/learning-powershell/README.md [hunk 1]
```diff
@@ -103,6 +103,7 @@ Note that all bash commands should continue working on PowerShell session.
 - [PowerShell 5 How-To Videos][script-guy-how-to] by Ed Wilson
 - [PowerShell Documentation](https://docs.microsoft.com/powershell)
 - [Interactive learning with PSKoans](https://aka.ms/pskoans)
+- [Windows PowerShell Cookbook: The Complete Guide to Scripting Microsoft's Command Shell ] (https://www.amazon.com/Windows-PowerShell-Cookbook-Scripting-Microsofts-ebook/dp/B00ARN9MEK)
```
Comment (vexx32) [inline docs/learning-powershell/README.md]:
```suggestion
- [Windows PowerShell Cookbook: The Complete Guide to Scripting Microsoft's Command Shell](https://www.amazon.com/Windows-PowerShell-Cookbook-Scripting-Microsofts-ebook/dp/B00ARN9MEK)
```
If you have a space between the closing bracket and the open parenthesis, the link doesn't format correctly. 🙂

**Review by vexx32 — CHANGES_REQUESTED** [review]

File: docs/learning-powershell/README.md (hunk 1 above)
Comment (SteveLee1287) [reply docs/learning-powershell/README.md]: Thank you very much, I did not catch that!!

**Comment by iSazonov** [comment]
@SteveL-MSFT Can you approve the PR?

**Comment by adityapatwardhan** [comment]
@joeyaiello Please review.

**Comment by SteveL-MSFT** [comment]
I believe all the current links are to free resources.  Even though this book was authored by a previous PowerShell Team member, I don't think we want to set a precedent and start advertising non-free resources.

**Comment by adityapatwardhan** [comment]
As per Steve's comment above, closing this PR. Please re-open if you do not agree.

**Pull request closed without merging**